packages/gatsby-plugin-utils/src/test-plugin-options-schema.ts

```diff
@@ -0,0 +1,35 @@
+import Joi from "joi"
```

Comment:
We need to use our custom version of Joi with the extensions we add!

```suggestion
import { Joi } from "./joi"
```